internal/config/add.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var AddCmd = &cobra.Command{
+	Use:              "add <account|contract|deployment|network>",
+	Short:            "Add resource to configuration",
+	Example:          "flow config add account",
+	ValidArgs:        []string{"account", "contract", "deployment", "network"},
```

Comment:
Is `ValidArgs` needed if all of these are subcommands?

Reply:
not anymore, before it was, forgot to remove